utilities/nodes/cpu_monitor/README.md

```
@@ -0,0 +1,141 @@
+ros2 launch cpu_monitor cpu_monitor_launch.xml \
+    csv_path:=/home/jetson/ws/log/cpu_test.csv \
+    sample_rate_hz:=1.0 top_n:=10
+```
+
+---
+
+## What gets logged
+
+### Console (every tick)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)